### ISSUE TITLE
internal CGroup metrics do not work on cgroupv2

### ISSUE THREAD

**Issue by EricJeffrey** [cockpit-project/cockpit#14420]
Cockpit version: cockpit-223-1.fc32.x86_64
OS: Fedora 32 (Workstation Edition)

CGroup Sampling seems to be not working on Fedora32. The prefix is no longer `/sys/fs/cgroup/memory` or `/sys/fs/cgroup/cpu` 
 as defined here [cockpitcgroupsamples.c](https://github.com/cockpit-project/cockpit/blob/2370dd7852dcbaf6e72617e1a185d010cf688663/src/bridge/cockpitcgroupsamples.c#L29), this is the dir list on my fedora32:
```
eric@localhost:~/coding/starter-kit$ ls -l /sys/fs/cgroup/
总用量 0
-r--r--r--.   1 root root 0  7月 28  2020 cgroup.controllers
-rw-r--r--.   1 root root 0  7月 27 21:33 cgroup.max.depth
-rw-r--r--.   1 root root 0  7月 27 21:33 cgroup.max.descendants
-rw-r--r--.   1 root root 0  7月 28  2020 cgroup.procs
-r--r--r--.   1 root root 0  7月 27 21:33 cgroup.stat
-rw-r--r--.   1 root root 0  7月 27 22:59 cgroup.subtree_control
-rw-r--r--.   1 root root 0  7月 27 21:33 cgroup.threads
-rw-r--r--.   1 root root 0  7月 27 21:33 cpu.pressure
-r--r--r--.   1 root root 0  7月 27 21:33 cpuset.cpus.effective
-r--r--r--.   1 root root 0  7月 27 21:33 cpuset.mems.effective
drwxr-xr-x.   2 root root 0  7月 27 21:59 init.scope
-rw-r--r--.   1 root root 0  7月 27 21:33 io.cost.model
-rw-r--r--.   1 root root 0  7月 27 21:33 io.cost.qos
-rw-r--r--.   1 root root 0  7月 27 21:33 io.pressure
drwxr-xr-x.   5 root root 0  7月 27 22:41 machine.slice
-rw-r--r--.   1 root root 0  7月 27 21:33 memory.pressure
drwxr-xr-x. 133 root root 0  7月 28 01:07 system.slice
drwxr-xr-x.   4 root root 0  7月 27 21:59 user.slice
```

I also tried to use starter-kit to fetch some cgroup message:
```js
testMetric() {
    var usageMetricsChannel = cockpit.metrics(1000, {
        source: "internal",
        metrics: [{ name: "cgroup.memory.usage", units: "bytes" }]
    });
    usageMetricsChannel.follow();
    usageMetricsChannel.onchanged = function () {
        console.log("onchanged, usageMetricsChannel: ", usageMetricsChannel);
    };
}
```
And I got nothing, not only the log, but all message in this channel:
![image](https://user-images.githubusercontent.com/33823695/88621395-e1e7dd00-d0d2-11ea-99a4-123b6c2a391f.png)

Also, it works find on my Ubuntu20, with cockpit215, but it's CGroupV1, having `/sys/fs/cgroup/memory` or `/cpu` dir.

**Comment by martinpitt** [comment]
Indeed this has never been ported to cgroupv2. We didn't notice because at least until recently docker didn't work on cgroupv2, and we don't even build cockpit-docker on Fedora/recent Debian/Ubuntu any more.

OOI, what do you need this for? Do you want to introduce podman container usage stats in cockpit-podman?

**Comment by EricJeffrey** [comment]
@martinpitt Well yeah, I just made a PR and found that it won't work on fedora32, [here](https://github.com/cockpit-project/cockpit-podman/pull/458).

**Comment by martinpitt** [comment]
I just implemented that in PR #14540
